nepal_compliance/api/api.py

```
@@ -19,37 +19,14 @@ def attendance(employee, status, shift, attendance_date, log_type, datetime, lat
-        attendance.insert()
-        frappe.db.commit()
+        if log_type in ['Check In', 'Check Out']:
+            attendance_doc = frappe.new_doc("Attendance")
+            attendance_doc.employee = employee
+            attendance_doc.status = status
+            attendance_doc.shift = shift
+            attendance_doc.attendance_date = attendance_date
+            attendance_doc.docstatus = 1
+            attendance_doc.insert()
+            frappe.db.commit()
     except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)